### DISCUSSION TITLE
I'm getting an error when run "mvn package" command. Can you please help me ?

### DISCUSSION THREAD

**Discussion by commodus** [datainsider-co/rocket-bi#87] [General]
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for datainsider:rocket-bi-server:jar:1.0-SNAPSHOT                                                                       
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.microsoft.sqlserver:mssql-jdbc:jar -> duplicate declaration of version 9.2.1.jre8 @ line 496, column 21
[WARNING]                                                                                                                                                                                           
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.                                                                                         
[WARNING]                                                                                                                                                                                           
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.                                                                                          
[WARNING]                                                                                                                                                                                           
[INFO]                                                                         
[INFO] --------------------< datainsider:rocket-bi-server >--------------------
[INFO] Building rocket-bi-server 1.0-SNAPSHOT                                  
[INFO]   from pom.xml                                                          
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- scrooge:20.4.0:compile (thrift-sources) @ rocket-bi-server ---
[INFO] finding thrift files in dependencies
[INFO] finding thrift files in referenced (reactor) projects                   
[INFO] No thrift files to compile.                                             
[INFO]                                                                         
[INFO] --- resources:3.3.1:resources (default-resources) @ rocket-bi-server ---
[WARNING] Using platform encoding (Cp1254 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] Copying 3 resources from conf to target\classes
[INFO] Copying 3 resources from sql to target\classes 
[INFO] Copying 106 resources from src\main\resources to target\classes
[INFO] 
[INFO] --- scala:3.2.0:add-source (scala-compile-first) @ rocket-bi-server ---
[INFO] Add Source directory: C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala
[INFO] Add Test Source directory: C:\TestRepos\rocket-bi\rocket-bi-server\src\test\scala
[INFO]
[INFO] --- scala:3.2.0:compile (scala-compile-first) @ rocket-bi-server ---
[WARNING]  Expected all dependencies to require Scala version: 2.12.12
[WARNING]  com.twitter:finatra-http_2.12:20.5.0 requires scala version: 2.12.8
[WARNING] Multiple versions of scala libraries detected!
[WARNING] Zinc server is not available at port 3030 - reverting to normal incremental compile
[INFO] Using incremental compilation
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  7.450 s
[INFO] Finished at: 2023-10-18T12:54:38+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.2.0:compile (scala-compile-first) on project rocket-bi-server: Execution scala-compile-first of goal net.alchim31.maven:scala-maven-plugin:3.2.0:compile failed
: Cannot invoke "java.lang.CharSequence.length()" because "this.text" is null -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

**Comment by tvc12** [comment]
Please check the Java version on your local machine:
```sh
java -version
```
This project requires **Java 8**.

**Reply by commodus** [reply]
Thank you for your quick response
![image](https://github.com/datainsider-co/rocket-bi/assets/1306778/e80a6c29-2fac-4bcf-842a-bba32d976a60)

**Reply by tvc12** [reply]
Can you provide version of maven?
```sh 
mvn -version
```

**Reply by commodus** [reply]
Yes, sure
![image](https://github.com/datainsider-co/rocket-bi/assets/1306778/491fc3c4-1f16-4d21-b04f-4b0d1d26dc79)

**Reply by tvc12** [reply]
Java used for Maven is Java 17. However, there is a problem. Please change it to Java 8.

**Reply by commodus** [reply]
![image](https://github.com/datainsider-co/rocket-bi/assets/1306778/1a2e7d25-0572-4327-a050-740baa6eaea1)

I changed it but I'm getting the "Java heap size" error. 

![image](https://github.com/datainsider-co/rocket-bi/assets/1306778/3e5c0b2d-048c-41c2-a6b0-35f42e411b68)


PS C:\TestRepos\rocket-bi\rocket-bi-server> mvn package
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for datainsider:rocket-bi-server:jar:1.0-SNAPSHOT
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.microsoft.sqlserver:mssql-jdbc:jar -> duplicate declaration of version 9.2.1.jre8 @ line 496, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] --------------------< datainsider:rocket-bi-server >--------------------
[INFO] Building rocket-bi-server 1.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- scrooge:20.4.0:compile (thrift-sources) @ rocket-bi-server ---
[INFO] finding thrift files in dependencies
[INFO] finding thrift files in referenced (reactor) projects
[INFO] No thrift files to compile.
[INFO]
[INFO] --- resources:3.3.1:resources (default-resources) @ rocket-bi-server ---
[WARNING] Using platform encoding (Cp1254 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] Copying 3 resources from conf to target\classes
[INFO] Copying 3 resources from sql to target\classes
[INFO] Copying 106 resources from src\main\resources to target\classes
[INFO] 
[INFO] --- scala:3.2.0:add-source (scala-compile-first) @ rocket-bi-server ---
[INFO] Add Source directory: C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala
[INFO] Add Test Source directory: C:\TestRepos\rocket-bi\rocket-bi-server\src\test\scala
[INFO]
[INFO] --- scala:3.2.0:compile (scala-compile-first) @ rocket-bi-server ---
[WARNING]  Expected all dependencies to require Scala version: 2.12.12
[WARNING]  com.twitter:finatra-http_2.12:20.5.0 requires scala version: 2.12.8
[WARNING] Multiple versions of scala libraries detected!
[WARNING] Zinc server is not available at port 3030 - reverting to normal incremental compile
[INFO] Using incremental compilation
[INFO] Compiling 864 Scala sources to C:\TestRepos\rocket-bi\rocket-bi-server\target\classes...
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\service\DashboardService.scala:38: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use updateSettings instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\service\DashboardService.scala:43: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use shareWithAnyone in ShareService")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\request\DirectoryRequest.scala:18: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated("use dashboardIds instead")
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\controller\http\DirectoryController.scala:264: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\service\DirectoryService.scala:56: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use remove instead of delete")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\chart\ChartSetting.scala:482: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("use TabControlChartSetting instead")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\chart\ChartSetting.scala:517: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("use TabControlChartSetting instead")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\chart\ChartSetting.scala:939: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use buildSankeyQuery instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\chart\Widget.scala:75: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated fontSize: String = "",
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\chart\Widget.scala:76: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated textColor: String = "",
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\chart\Widget.scala:77: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated backgroundColor: String = "",
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\chart\Widget.scala:78: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated isHtmlRender: Boolean,
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\domain\chart\Widget.scala:93: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("unused widget")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\schema\repository\DDLExecutor.scala:54: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use dropTable with table type instead of")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\schema\repository\DDLExecutor.scala:114: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("Method is supported only in clickhouse engine")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\repository\DashboardRepository.scala:30: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use update instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\repository\SchemaManager.scala:32: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("use MySqlSchemaManager instead") // TODO: migrate this class to MySqlSchemaManager
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\service\CustomJobFactory.scala:17: non-variable type argument co.datainsider.bi.service.BoostJob in type Class[co.datainsider.bi.service.BoostJob] (t
he underlying of Class[co.datainsider.bi.service.BoostJob]) is unchecked since it is eliminated by erasure
[WARNING]       if (jobClass.isInstanceOf[Class[BoostJob]]) {
[WARNING]                                ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\bi\service\DashboardService.scala:369: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("will remove when remove share")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\user_profile\service\RegistrationService.scala:22: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("unused code, cause unsupported manual register use register in admin controller instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\admin\service\AdminUserService.scala:50: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated(message = "Use searchUsersV2 instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\admin\filters\users\BaseUserFilter.scala:5: imported `PermissionResult' is permanently hidden by definition of object PermissionResult in package u
sers
[WARNING] import datainsider.client.domain.permission.{PermissionResult, Permitted, UnPermitted}
[WARNING]                                              ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\admin\filters\users\ActivateAccessFilterImpl.scala:7: imported `PermissionResult' is permanently hidden by definition of object PermissionResult in
 package users
[WARNING] import datainsider.client.domain.permission.PermissionResult
[WARNING]                                             ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\admin\service\AdminUserService.scala:132: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated(message = "Use searchUsersV2 instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\user_profile\service\UserProfileService.scala:54: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("Use SearchUsers with keywords instead of")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\login_provider\repository\OAuthProvider.scala:101: non-variable type argument String in type pattern scala.collection.immutable.Map[String,String] 
(the underlying of Map[String,String]) is unchecked since it is eliminated by erasure
[WARNING]       case Some(s: Map[String, String]) =>
[WARNING]                    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\login_provider\repository\OAuthProvider.scala:169: non-variable type argument String in type pattern scala.collection.immutable.Map[String,String] 
(the underlying of Map[String,String]) is unchecked since it is eliminated by erasure
[WARNING]         case Some(s: Map[String, String]) => s
[WARNING]                      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\login_provider\service\OAuthService.scala:24: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("use OAuthServiceProviderV2 ")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\user_caas\repository\UserPasswordRealm.scala:103: This catches all Throwables. If this is really intended, use `case e : Throwable` to clear this w
arning.
[WARNING]       case e => throw new AuthenticationException(e)
[WARNING]            ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\user_profile\controller\http\request\OrganizationRequest.scala:28: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("does not support create org with domain")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\user_profile\service\RegistrationService.scala:27: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("unused code, cause change flow forgot password")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\caas\user_profile\util\Utils.scala:62: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use isValidEmailV1, this method maybe incorrect")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\service\ShareETLService.scala:35: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use listSharedEtlJobs in EtlJobService instead of")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\service\ETLPreviewService.scala:43: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("will remove in next release")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\domain\ETLStatus.scala:11: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use Running instead of")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\domain\ETLStatus.scala:13: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use Done instead of")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\domain\EtlJob.scala:41: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated("use operatorInfo instead of")
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\pipeline\operator\GetOperator.scala:17: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated("don't use columns in this field, cause table schema can be changed")
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\domain\operator\OldJoinOperator.scala:38: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("Use JoinOperator instead")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\domain\operator\OldPythonOperator.scala:13: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("use PythonOperator instead")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\domain\operator\SendToGroupEmailOperator.scala:22: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("use SendGroupEmailOperator instead")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\domain\persist\JdbcPersistConfiguration.scala:9: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("use JDBCPersistOperator instead")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\pipeline\operator\OperatorService.scala:322: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use getDbName in implement of EtlTableService instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\pipeline\operator\persist\JdbcPersistOperatorExecutor.scala:100: A try without a catch or finally is equivalent to putting its body in a block;
 no exceptions are handled.
[WARNING]         try {
[WARNING]         ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\service\ETLPreviewService.scala:217: non-variable type argument co.datainsider.datacook.pipeline.operator.TableResultOperator in type pattern O
ption[co.datainsider.datacook.pipeline.operator.TableResultOperator] is unchecked since it is eliminated by erasure
[WARNING]         case operator: Option[TableResultOperator] => operator.get.destTableConfiguration.tblName
[WARNING]                        ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\datacook\service\ETLService.scala:75: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("unused method")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobscheduler\service\JobService.scala:36: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use multiCreate with jobs instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobscheduler\domain\job\Job.scala:156: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use multiCreate in JobService instead of")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobscheduler\domain\job\GaJob.scala:39: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated("use GaSource instead of") accessToken: String = "",
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobscheduler\domain\job\GaJob.scala:40: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated("use GaSource instead of") refreshToken: String = "",
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobscheduler\domain\job\GoogleSheetJob.scala:16: @deprecated now takes two arguments; see the scaladoc.
[WARNING] @deprecated("use GoogleSheetJobV2 instead")
[WARNING]  ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobscheduler\domain\job\Job.scala:179: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("use GoogleSheetsV2 instead")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobworker\service\MetadataService.scala:51: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("method is unused, need check before remove")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobworker\service\MetadataService.scala:54: @deprecated now takes two arguments; see the scaladoc.
[WARNING]   @deprecated("method is unused, need check before remove")
[WARNING]    ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobworker\domain\job\GaJob.scala:29: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated("use GaSource instead of") accessToken: String = "",
[WARNING]      ^
[WARNING] C:\TestRepos\rocket-bi\rocket-bi-server\src\main\scala\co\datainsider\jobworker\domain\job\GaJob.scala:30: @deprecated now takes two arguments; see the scaladoc.
[WARNING]     @deprecated("use GaSource instead of") refreshToken: String = ""
[WARNING]      ^
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  55.287 s
[INFO] Finished at: 2023-10-18T15:29:54+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Java heap space -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/OutOfMemoryError

**Reply by tvc12** [reply]
Checkout increase heap size for maven in here https://subscription.packtpub.com/book/cloud-and-networking/9781783986767/1/ch01lvl1sec09/configuring-the-heap-size
and here
https://cwiki.apache.org/confluence/plugins/servlet/mobile?contentId=4588986#content/view/4588986

Btw, you run cmd without test:
```sh
mvn package -DskipTests
```
